### ISSUE TITLE
rfe: (Results) Indicate wether a position is 'shared' or not

### ISSUE THREAD

**Issue by bremco** [JammerCore/JammerCore#1590]
If a category-score matches exactly between multiple games in the same event/type, they share a position. Famously, the Jam portion of LD37 had [3 overall winners](http://ludumdare.com/compo/ludum-dare-37/).

While this in itself may not be a problem, it may 'mislead' people. For example, [Wikipedia](https://en.wikipedia.org/wiki/Ludum_Dare#cite_ref-48) only lists one of those three winners of the LD37 Jam. Also, people that check their own results, then see someone else claiming the exact same result, might incorrectly suspect that something is wrong unless they at least know about the possibility of shared positions.

**Comment by mikekasprzak** [comment]
Yes, mentioning this is a tie would be useful. Not quite sure the right way
to implement it, but it's something to think about.

In Wikipedia's case, it sounds like that should be corrected on Wikipedia.
😉

**Comment by bremco** [comment]
... allright, I've updated Wikipedia with the 3-way tie 🙂

**Comment by mikekasprzak** [comment]
Great. Thanks for making the change. :smile:
